.rubocop.yml

```diff
@@ -0,0 +1,77 @@
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  TargetRubyVersion: 3.1.0
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'vendor/**/*'
+    - 'script/**/*'
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'spec/vcr/**/*'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+
+Layout/LineLength:
+  Max: 120
+  Exclude:
+    - config/**/*
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Alias:
+  Enabled: false
+
+Style/HashLikeCase:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/LetSetup:
+  Enabled: false
+
+Rails/LinkToBlank:
```

Comment:
just FYI: there used to be [this vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/), but modern browsers now add it by default, so this cop is obsolete now. No need to change anything here, just quick lesson to learn :)